store.html

```diff
@@ -86,9 +86,15 @@ <h1 class="order-heading">Order your Karma wifi<br/>device today!</h1>
                                     <!-- Create a placeholder -->
                                     <option value="" disabled selected hidden><em>Select your city...</em></option>
 
```

Comment:
Really nice use of <em> here, i ending up using CSS here as I could not get <em> to work.